sdk/keyvault/keyvault-keys/src/keysModels.ts

```diff
@@ -612,3 +612,78 @@ export interface GetCryptographyClientOptions {
    */
   keyVersion?: string;
 }
+
+/**
+ * Options for {@link KeyClient.rotateKey}
+ */
+export interface RotateKeyOptions extends coreHttp.OperationOptions {}
+
+/**
+ * The properties of a key rotation policy that the client can set for a given key.
+ */
+export interface KeyRotationPolicyProperties {
+  /**
+   * The expiry time of the policy that will be applied on new key versions, defined as an ISO 8601 duration.
+   */
+  expiresIn?: string;
```

Comment:
Hmm in other places we have `expiresOn`, like in the AccessToken of core-auth. Would `expiresOn` be better for consistency?

Reply:
In this case no - expiresIn makes more sense as it's a relative duration not a date

Reply:
ok!